Checkout fails on windows-latest when using @master

Builds are failing on windows. Logs attached.

```
2019-12-03T16:15:23.3178351Z ##[section]Starting: Request a runner to run this job
2019-12-03T16:15:24.2697462Z Requesting a hosted runner in current repository's account/organization with labels: 'windows-latest', require runner match: True
2019-12-03T16:15:24.3347571Z Labels matched hosted runners has been found, waiting for one of them get assigned for this job.
2019-12-03T16:15:24.3834888Z ##[section]Finishing: Request a runner to run this job
2019-12-03T16:15:31.7862022Z Current runner version: '2.161.0'
2019-12-03T16:15:31.7863222Z Prepare workflow directory
2019-12-03T16:15:31.8356961Z Prepare all required actions
2019-12-03T16:15:31.8391209Z Download action repository 'actions/checkout@master'
2019-12-03T16:15:32.7930286Z Download action repository 'actions/setup-python@master'
2019-12-03T16:15:33.7783103Z Download action repository 'actions/upload-artifact@master'
2019-12-03T16:15:34.1439691Z ##[group]Run actions/checkout@master
2019-12-03T16:15:34.1440081Z with:
2019-12-03T16:15:34.1440360Z   repository: arlyon/graphics
2019-12-03T16:15:34.1440572Z   token: ***
2019-12-03T16:15:34.1440704Z   clean: true
2019-12-03T16:15:34.1440818Z   fetch-depth: 1
2019-12-03T16:15:34.1440948Z   lfs: false
2019-12-03T16:15:34.1441076Z ##[endgroup]
2019-12-03T16:15:35.1466185Z Added matchers: 'checkout-git'. Problem matchers scan action output for known warning or error strings and report these inline.
2019-12-03T16:15:35.1467998Z Syncing repository: arlyon/graphics
2019-12-03T16:15:35.1468618Z Working directory is 'd:\a\graphics\graphics'
2019-12-03T16:15:35.1568360Z [command]"C:\Program Files\Git\bin\git.exe" version
2019-12-03T16:15:35.1768829Z git version 2.23.0.windows.1
2019-12-03T16:15:35.2022792Z Removed matchers: 'checkout-git'
2019-12-03T16:15:35.2024723Z ##[error]Command failed: rd /s /q "d:\a\graphics\graphics"
The process cannot access the file because it is being used by another process.

2019-12-03T16:15:35.2106752Z ##[error]Node run failed with exit code 1
2019-12-03T16:15:35.2147889Z Cleaning up orphan processes
```

Changing the version to @v1 seems to resolve the problem.

--- a/src/git-source-provider.ts
+++ b/src/git-source-provider.ts
@@ -61,6 +61,7 @@ export async function getSource(settings: ISourceSettings): Promise<void> {
   ) {
     // Delete the contents of the directory. Don't delete the directory itself
     // since it may be the current working directory.
+    core.info(`Deleting the contents of '${settings.repositoryPath}'`)
     for (const file of await fs.promises.readdir(settings.repositoryPath)) {
       await io.rmRF(path.join(settings.repositoryPath, file))
     }

--- a/src/git-source-provider.ts
+++ b/src/git-source-provider.ts
@@ -59,8 +59,11 @@ export async function getSource(settings: ISourceSettings): Promise<void> {
       settings.clean
     ))
   ) {
-    await io.rmRF(settings.repositoryPath)
-    await io.mkdirP(settings.repositoryPath)
+    // Delete the contents of the directory. Don't delete the directory itself
+    // since it may be the current working directory.
+    for (const file of await fs.promises.readdir(settings.repositoryPath)) {
+      await io.rmRF(settings.repositoryPath)
+    }
   }
 
   // Initialize the repository

--- a/src/git-source-provider.ts
+++ b/src/git-source-provider.ts
@@ -62,7 +62,7 @@ export async function getSource(settings: ISourceSettings): Promise<void> {
     // Delete the contents of the directory. Don't delete the directory itself
     // since it may be the current working directory.
     for (const file of await fs.promises.readdir(settings.repositoryPath)) {
-      await io.rmRF(settings.repositoryPath)
+      await io.rmRF(path.join(settings.repositoryPath, file))
     }
   }
 

